### PR TITLE
feat: Added meta tags for referencing

### PR DESCRIPTION
**Pull request by frgfm**
This PR adds meta tags for proper SEO optimization.

### CODE REVIEW

**Comment by fe51** [comment]
Hey @frgfm, 
Thank you for adding keywords to metadata !

File: _config.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: Pyronear
 description: "Détection précoce de feux de forêt" # this means to ignore newlines until "baseurl:"
 baseurl: "https://pyronear.github.io/" # the subpath of your site, e.g. /blog/
 url: "" # the base hostname & protocol for your site
+tags: [pyronear,association,deeplearning,feux,incendies,raspberry,dataforgood,atraksis]
```
Comment (fe51) [inline _config.yml:6]:
```suggestion
tags: [pyronear,association,deeplearning,feux,incendies,détection,forêt,raspberry,dataforgood,atraksis]
```

Even if these words are in description, shouldn't we add "forêt" and "détection" in keywords tags ?

**Review by fe51 — APPROVED** [review]

**Comment by fe51** [comment]
In fact, I realize, from various sources, meta keywords are not used for referencing, especially by google (since 2009)
Hence, we migth need another approach to improve Pyronear referecing. 

https://webmasters.googleblog.com/2009/09/google-does-not-use-keywords-meta-tag.html
https://seo-trench.com/balise-meta-keywords-wordpress
https://quinchyriya.com/insights/seo/plus-utiliser-balise-meta-keywords-2020/

Because the meta description is quite short (30 characters), a solution to improve referencing might be to enlarge it with.. some keywords. https://support.google.com/webmasters/answer/35624

**Comment by frgfm** [comment]
> In fact, I realize, from various sources, meta keywords are not used for referencing, especially by google (since 2009)
> Hence, we migth need another approach to improve Pyronear referecing.
> 
> https://webmasters.googleblog.com/2009/09/google-does-not-use-keywords-meta-tag.html
> https://seo-trench.com/balise-meta-keywords-wordpress
> https://quinchyriya.com/insights/seo/plus-utiliser-balise-meta-keywords-2020/
> 
> Because the meta description is quite short (30 characters), a solution to improve referencing might be to enlarge it with.. some keywords. https://support.google.com/webmasters/answer/35624

I had doubts and thought we should add it by default, but you're correct! I'll close the PR, #8 should be more relevant!

**Pull request closed without merging**